### ISSUE TITLE
Unexpected out of memory error in hybrid nanopore assembly

### ISSUE THREAD

**Issue by maleasy** [ablab/spades#173]
Hi,

I am running a assembly with illumina and nanopore data. The Assembly of the Illumina data works fine, but at the first Nanopore step (Aligning long reads with bwa-mem based aligner) I get a out of memory error.

I provided 350 GB memory with `-m`, the log says that 118 GB are used, so there should be no problem. When looking at the system log and memory usage stats on the server, there is no indication that the memory was full.

I reran with `--continue`, to the same result.

```
  9:29:17.735    67G / 118G  INFO    General                 (hybrid_aligning.cpp       : 288)   Aligning long reads with bwa-mem based aligner
<jemalloc>: Error in calloc(): out of memory


== Error ==  system call for: "['spades-core', 'metaspades_assembly_results/K127/configs/config.info', 'metaspades_assembly_results/K127/configs/mda_mode.info', 'metaspades_assembly_results/K127/configs/meta_mode.info']" finished abnormally, err code: -6

```

[metaspades_continue.log](https://github.com/ablab/spades/files/2375232/metaspades_continue.log)
[metaspades.log](https://github.com/ablab/spades/files/2375233/metaspades.log)
[params.txt](https://github.com/ablab/spades/files/2375234/params.txt)

**Comment by asl** [comment]
I believe it's the problem we already fixed and the fix will be in SPAdes 3.12.1

**Comment by maleasy** [comment]
Hi Anton,

thanks for your quick answer. 
It is possible to get a nightly/beta version of 3.12.1?

Best,
Chris

**Comment by asl** [comment]
Fixed in 3.13.0
